### PR TITLE
Merge pull request #5 from aashna15sethi/main

### PR DESCRIPTION
**Pull request by aashna15sethi**
Update 01-coins.js

### CODE REVIEW

**Comment by JeremyIV** [comment]
Your use of branches and pull requests seems kind of messy... you're supposed to finish your homework all on one branch, and then create a pull request for the whole thing at once. This sequence of PRs is hard to follow.

**Comment by aashna15sethi** [comment]
> Your use of branches and pull requests seems kind of messy... you're supposed to finish your homework all on one branch, and then create a pull request for the whole thing at once. This sequence of PRs is hard to follow.

I apologize for that. I initially committed the changes and merged them with the main branch while forgetting to keep the PR open. So I had to delete the files and recreate the PR with the newly changed files. I will keep these things in my mind in the future. Thank you for your feedback.
